### PR TITLE
zh-cn: synchronize the translation of `Blob.slice()`

### PR DESCRIPTION
**Pull request by Yukiniro**
### Description

en-US: https://developer.mozilla.org/en-US/docs/Web/API/Blob/slice
zh-CN: https://developer.mozilla.org/zh-CN/docs/Web/API/Blob/slice

### CODE REVIEW

File: files/zh-cn/web/api/blob/slice/index.md [hunk 1]
```diff
@@ -1,32 +1,34 @@
 ---
-title: Blob.slice
+title: Blob: slice() 方法
+short-title: slice()
```
Comment (yin1999) [inline files/zh-cn/web/api/blob/slice/index.md:3]: Please remove the [short-title](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E5%85%83%E6%95%B0%E6%8D%AE)

```suggestion
title: Blob：slice() 方法
```

Comment (Yukiniro) [reply files/zh-cn/web/api/blob/slice/index.md:3]: ok

File: files/zh-cn/web/api/blob/slice/index.md [hunk 2]
```diff
@@ -1,32 +1,33 @@
...
 
 ### 参数
 
-- `start` {{ optional_inline() }}
-  - : 这个参数代表 {{domxref("Blob")}} 里的下标，表示第一个会被会被拷贝进新的 {{domxref("Blob")}} 的字节的起始位置。如果你传入的是一个负数，那么这个偏移量将会从数据的末尾从后到前开始计算。举例来说，-10 将会是 {{domxref("Blob")}} 的倒数第十个字节。它的默认值是 0，如果你传入的 start 的长度大于源 {{domxref("Blob")}} 的长度，那么返回的将会是一个长度为 0 并且不包含任何数据的一个 {{domxref("Blob")}} 对象。
+- `start` {{optional_inline()}}
+  - : 这个参数代表 {{domxref("Blob")}} 里的下标，表示第一个会被会被拷贝进新的 {{domxref("Blob")}} 的字节的位置。如果你传入的是一个负数，那么这个偏移量将会从数据的末尾从后到前开始计算。例如，-10 将会是 {{domxref("Blob")}} 的倒数第十个字节。它的默认值是 0，如果你传入的 `start` 的长度大于源 {{domxref("Blob")}} 的长度，那么返回的将会是一个长度为 0 并且不包含任何数据的一个 {{domxref("Blob")}} 对象。
 - `end` {{ optional_inline() }}
-  - : 这个参数代表的是 {{domxref("Blob")}} 的一个下标，这个下标 -1 的对应的字节将会是被拷贝进新的{{domxref("Blob")}} 的最后一个字节。如果你传入了一个负数，那么这个偏移量将会从数据的末尾从后到前开始计算。举例来说， -10 将会是 {{domxref("Blob")}} 的倒数第十个字节。它的默认值就是它的原始长度 (`size`).
+  - : 这个参数代表 {{domxref("Blob")}} 里的下标，表示第一个 \*不会\*被拷贝进新的 {{domxref("Blob")}} 的字节的位置（换句话说，这个位置的字节不会被拷贝）。如果你传入了一个负数，那么这个偏移量将会从数据的末尾从后到前开始计算。举例来说， -10 将会是 {{domxref("Blob")}} 的倒数第十个字节。它的默认值就是它的原始长度 `size`.
```
Comment (yin1999) [inline files/zh-cn/web/api/blob/slice/index.md:24]:
```suggestion
  - : 这个参数代表 {{domxref("Blob")}} 里的第一个*不会*被拷贝进新的 {{domxref("Blob")}} 的字节的索引（换句话说，这个索引的字节不会被拷贝）。如果你传入了一个负数，那么这个偏移量将会从数据的末尾从后到前开始计算。举例来说，-10 将会是 {{domxref("Blob")}} 的倒数第十个字节。它的默认值就是它的原始长度 `size`。
```

File: files/zh-cn/web/api/blob/slice/index.md [hunk 3]
```diff
@@ -1,32 +1,33 @@
...
+slice(start, end)
+slice(start, end, contentType)
 ```
 
 ### 参数
 
-- `start` {{ optional_inline() }}
-  - : 这个参数代表 {{domxref("Blob")}} 里的下标，表示第一个会被会被拷贝进新的 {{domxref("Blob")}} 的字节的起始位置。如果你传入的是一个负数，那么这个偏移量将会从数据的末尾从后到前开始计算。举例来说，-10 将会是 {{domxref("Blob")}} 的倒数第十个字节。它的默认值是 0，如果你传入的 start 的长度大于源 {{domxref("Blob")}} 的长度，那么返回的将会是一个长度为 0 并且不包含任何数据的一个 {{domxref("Blob")}} 对象。
+- `start` {{optional_inline()}}
+  - : 这个参数代表 {{domxref("Blob")}} 里的下标，表示第一个会被会被拷贝进新的 {{domxref("Blob")}} 的字节的位置。如果你传入的是一个负数，那么这个偏移量将会从数据的末尾从后到前开始计算。例如，-10 将会是 {{domxref("Blob")}} 的倒数第十个字节。它的默认值是 0，如果你传入的 `start` 的长度大于源 {{domxref("Blob")}} 的长度，那么返回的将会是一个长度为 0 并且不包含任何数据的一个 {{domxref("Blob")}} 对象。
```
Comment (yin1999) [inline files/zh-cn/web/api/blob/slice/index.md:22]:
```suggestion
  - : 这个参数代表 {{domxref("Blob")}} 里的第一个会被拷贝进新的 {{domxref("Blob")}} 的字节的索引。如果你传入的是一个负数，那么这个偏移量将会从数据的末尾从后到前开始计算。例如，-10 将会是 {{domxref("Blob")}} 的倒数第十个字节。它的默认值是 0。如果你传入的 `start` 的长度大于源 {{domxref("Blob")}} 的长度，那么返回的将会是一个长度为 0 并且不包含任何数据的一个 {{domxref("Blob")}} 对象。
```

File: files/zh-cn/web/api/blob/slice/index.md [hunk 4]
```diff
@@ -1,32 +1,33 @@
...
 - `end` {{ optional_inline() }}
-  - : 这个参数代表的是 {{domxref("Blob")}} 的一个下标，这个下标 -1 的对应的字节将会是被拷贝进新的{{domxref("Blob")}} 的最后一个字节。如果你传入了一个负数，那么这个偏移量将会从数据的末尾从后到前开始计算。举例来说， -10 将会是 {{domxref("Blob")}} 的倒数第十个字节。它的默认值就是它的原始长度 (`size`).
+  - : 这个参数代表 {{domxref("Blob")}} 里的下标，表示第一个 \*不会\*被拷贝进新的 {{domxref("Blob")}} 的字节的位置（换句话说，这个位置的字节不会被拷贝）。如果你传入了一个负数，那么这个偏移量将会从数据的末尾从后到前开始计算。举例来说， -10 将会是 {{domxref("Blob")}} 的倒数第十个字节。它的默认值就是它的原始长度 `size`.
 - `contentType` {{ optional_inline() }}
-  - : 给新的 {{domxref("Blob")}} 赋予一个新的文档类型。这将会把它的 type 属性设为被传入的值。它的默认值是一个空的字符串。
+  - : 给新的 {{domxref("Blob")}} 赋予一个新的文档类型。这将会把它的 `type` 属性设为被传入的值。它的默认值是一个空的字符串。
```
Comment (yin1999) [inline files/zh-cn/web/api/blob/slice/index.md:26]:
```suggestion
  - : 给新的 {{domxref("Blob")}} 赋予一个新的内容类型。这将会把它的 `type` 属性设为被传入的值。它的默认值是一个空字符串。
```

File: files/zh-cn/web/api/blob/slice/index.md [hunk 5]
```diff
@@ -1,32 +1,33 @@
...
 
 ### 返回值
 
-一个新的 {{domxref("Blob")}} 对象，它包含了原始 {{domxref("Blob")}} 对象的某一个段的数据。
+一个新的 {{domxref("Blob")}} 对象，它包含了原始 {{domxref("Blob")}} 对象的某一个段的数据。原始的 {{domxref("Blob")}} 不会被改变。
```
Comment (yin1999) [inline files/zh-cn/web/api/blob/slice/index.md:30]:
```suggestion
一个新的 {{domxref("Blob")}} 对象，它包含了调用该方法的原始 blob 对象所包含数据的指定子集。原始的 blob 不会被改变。
```

**Review by yin1999 — COMMENTED** [review]
let's align the translation with the english document.

**Review by yin1999 — APPROVED** [review]
Thanks @Yukiniro

**Pull request merged**